src/main/java/com/example/springfirsthw/dto/CreateBookRequestDto.java

```diff
@@ -0,0 +1,24 @@
+package com.example.springfirsthw.dto;
+
+import jakarta.validation.constraints.NotNull;
+import jakarta.validation.constraints.Positive;
+import jakarta.validation.constraints.Size;
+import java.math.BigDecimal;
+import lombok.Data;
+
+@Data
+public class CreateBookRequestDto {
+    @NotNull(message = "can't be null")
```

Comment:
It would be more informative to write what exactly cannot be null

Reply:
He adds info about field name in exception handler, it seems to be ok.